### PR TITLE
CRS-1163 ARD & SED for determinate sentences less than 12 months wher…

### PR DESCRIPTION
**Pull request by jbowdidge-moj**
…e offender is less than 18 at end of custodial period

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/exceptions/SentenceIdentificationException.kt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.exceptions
+
+class SentenceIdentificationException(msg: String) : Exception(msg)
```
Comment (ldlharper) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/exceptions/SentenceIdentificationException.kt:3]: Is this used?

**Review by ldlharper — COMMENTED** [review]
LGTM. Just 1 minor comment.

File: src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/exceptions/SentenceIdentificationException.kt (hunk 1 above)
Comment (jbowdidge-moj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/exceptions/SentenceIdentificationException.kt]: Done

**Review by ldlharper — APPROVED** [review]
LGTM

**Pull request merged**